### DISCUSSION TITLE
Collection database not being updated for some songs

### DISCUSSION THREAD

**Discussion by radstevee** [ppy/osu#19072] [Q&A] [answered]
I'm not sure if I'm right here, but for example for the song `Toby Fox - THE WORLD REVOLVING (mapped by Fatfan Kolek)` isn't being updated. I got an S in Normal 2 times, but the highscore still shows A. It even shows the S in the recent plays: 
![image](https://user-images.githubusercontent.com/96977790/178249254-aa98dccc-1d94-4381-8807-c3eb62452e74.png)

**Comment by peppy** [comment, marked as answer]
Sounds like an osu! stable question. Please ask accounts@ppy.sh instead of posting here.

**Reply by radstevee** [reply]
Would asking on [here](https://github.com/ppy/osu-stable-issues) also work?

**Reply by peppy** [reply]
No.
